src/Plugin/WebformHandler/OrgunitWebformHandler.php

```diff
@@ -53,10 +53,10 @@ public function alterForm(
 
       if ($externals) {
         $external_options = [];
-        foreach ($externals as $username => $e) {
-          // Get ID from username.
-          $user = user_load_by_name($username);
-          $external_options[$user->id()] = $username;
+        foreach ($externals as $external) {
```

Comment:
Why this change?

Reply:
Becuase having the initials as the user key for engagements/engagement associations is specific business logic in our test data and it will not necessarily always be the case. It's safer to get users by their GIR UUID because it'll always work regardless of the user key used.

Reply:
OK, makes sense.